http/src/test/java/com/redhat/lightblue/client/http/LightblueHttpClientTest.java

```diff
@@ -273,6 +275,50 @@ public boolean processDocument(ResultStream.StreamDoc doc) {
             
         }
     }
+    
+    private static final String diagnosticsResponse = 
+            "{\"MongoCRUDController\":"
+            + "{\"healthy\":true,\"message\":\""
+            + "[Mongo Config [lightbluemongo1.dev.a1.vary.redhat.com:27017, DatabaseName: metadata]=>ping:OK, "
+            + "Mongo Config [lightbluemongo1.dev.a1.vary.redhat.com:27017, DatabaseName: data]=>ping:OK]\"},"
+            + "\"ldap-auth-healthcheck\":{\"healthy\":true,"
+            + "\"message\":\"LDAPConnection [DN: uid=lightblueapp,ou=serviceusers,ou=lightblue,dc=redhat,dc=com, "
+            + "Status: true]\"}}";
+   
+    @Test
+    public void testLightblueDiagnosticsElementPresent() throws Exception {
+        
+        DiagnosticsElement expectedElementDiagnostics = new DiagnosticsElement("MongoCRUDController", true, "[Mongo Config [lightbluemongo1.dev.a1.vary.redhat.com:27017, "
+                + "DatabaseName: metadata]=>ping:OK, Mongo Config [lightbluemongo1.dev.a1.vary.redhat.com:27017, DatabaseName: data]=>ping:OK]");
+        
+        when(httpTransport.executeRequest(any(LightblueRequest.class), anyString())).thenReturn(new FakeResponse(diagnosticsResponse, null));
```

Comment:
When using mocks I think it's a good idea to make sure the right parameters are being passed as well. So for example we should probably test the base URI is correct.

We also need a unit test for the diagnostics request itself. There are others like it in lightblue-client/core/src/test/java/com/redhat/lightblue/client/request .